app/controllers/help_controller.rb

```diff
@@ -1,4 +1,10 @@
 class HelpController < ApplicationController
-	before_filter -> { @css_framework = :bootstrap }
-	layout ->(c) { request.format == :mobile ? "application" : "with_header_with_footer" }
-end
+
+  before_filter -> { @css_framework = :bootstrap }
+  layout -> (c) { request.format == :mobile ? "application" : "with_header_with_footer" }
+
+  def faq
+    @data_chat = AppConfig.chat.enabled ? "enabled" : "disabled"
```

Comment:
It's already available and solves the problem better than your current solution. I don't see why that would be over-engineered, in fact it's less code you have to add in this PR.